WebContent/js/components/JobInstance.js

```diff
@@ -95,14 +95,18 @@ class JobInstance extends React.Component {
         const statusStyleActive = { display: 'inline' };
         const statusStyleAbend = { color: 'red', display: 'inline' };
         const statusStyleComplete = { color: 'grey', display: 'inline' };
-        if (job.get('returnCode')) {
-            if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')) {
-                return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+        if (job.get('status').toLowerCase().includes('output')) {
+            if (job.get('returnCode')) {
+                if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')
```

Comment:
There's a lot of repetition here, both in terms of string comparison and converting return code to lower case.
I think this would read better if we create a lowerCaseReturnCode variable which we use for the comparison. 

As the set of error case strings has increased to 4 I think we should store these in a collection now, and the if condition should be checking if the return code is included in the collection (One way would be to construct an array of strings and then using array.find to determine if the return code is in the known error case strings), this will make expanding the set of error case strings much easier. I've actually just come across another error case string, "sec error" so please add this to the collection too.

Reply:
Agreed.
I will 1) Create a variable for lower case return code.
         2) Add an Array to store existing 4 error case plus "sec error".
Thanks.